atproto/auth/oauth/util.go

```diff
@@ -0,0 +1,23 @@
+package oauth
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/base64"
+)
+
+// this generates pseudo-unique nonces to prevent token (JWT) replay. these do not need to be cryptographically resilient
```

Comment:
> these do not need to be cryptographically resilient

This is no longer correct since `randomNonce()` is also used for state+pkce (see my other comment)